src/commands/codepush/lib/legacy-codepush-service-client.ts

```diff
@@ -0,0 +1,108 @@
+import * as Request from "request";
```

Comment:
Nitpick: Should be imported as `request` not `Request`. Uppercase is for class constructors.

Reply:
fixed